openpype/hosts/maya/plugins/create/create_unreal_staticmesh.py

```diff
@@ -14,20 +13,14 @@ class CreateUnrealStaticMesh(plugin.Creator):
     icon = "cube"
     dynamic_subset_keys = ["asset"]
 
-    def __init__(self, *args, **kwargs):
-        """Constructor."""
-        super(CreateUnrealStaticMesh, self).__init__(*args, **kwargs)
-        self._project_settings = get_project_settings(
-            legacy_io.Session["AVALON_PROJECT"])
-
     @classmethod
     def get_dynamic_data(
-            cls, variant, task_name, asset_id, project_name, host_name
+        cls, variant, task_name, asset_id, project_name, host_name
     ):
         dynamic_data = super(CreateUnrealStaticMesh, cls).get_dynamic_data(
             variant, task_name, asset_id, project_name, host_name
         )
-        dynamic_data["asset"] = legacy_io.Session.get("AVALON_ASSET")
+        dynamic_data["asset"] = get_current_asset_name()
```

Comment:
Same question as above. Isn't this data in the `create_context`? Or isn't that available during `get_dynamic_data`?

Reply:
Same answer.